tests/PHPStan/Levels/data/acceptTypes-5.json

```diff
@@ -59,11 +59,6 @@
         "line": 144,
         "ignorable": true
     },
-    {
```

Comment:
Why was this error removed?

Reply:
Don't know. It was removed when I launched phing locally. I thought it was expected.

Reply:
It means that this error stopped being reported. It's not something you should commit, but instead fix so that's continued being reported. I have no idea why - probably SimpleXMLElement or ResourceBundle has `__toString` method or something like that.

Reply:
Found it.

It's because of this [`if` block](https://github.com/phpstan/phpstan/blob/master/src/Type/ObjectType.php#L85). Not sure how to fix this without breaking other parts.

Any hints @ondrejmirtes ?